<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>820</width>
    <height>933</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>820</width>
    <height>930</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="formLayoutWidget">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>781</width>
      <height>861</height>
     </rect>
    </property>
    <layout class="QFormLayout" name="formLayout">
     <property name="sizeConstraint">
      <enum>QLayout::SetMinAndMaxSize</enum>
     </property>
     <property name="fieldGrowthPolicy">
      <enum>QFormLayout::ExpandingFieldsGrow</enum>
     </property>
     <property name="labelAlignment">
      <set>Qt::AlignJustify|Qt::AlignVCenter</set>
     </property>
     <property name="formAlignment">
      <set>Qt::AlignCenter</set>
     </property>
     <property name="leftMargin">
      <number>0</number>
     </property>
     <property name="rightMargin">
      <number>0</number>
     </property>
     <item row="0" column="1">
      <widget class="QLabel" name="MAIN_TEXT">
       <property name="enabled">
        <bool>true</bool>
       </property>
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="font">
        <font>
         <family>IM혜민 Bold</family>
         <pointsize>20</pointsize>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>인스타크롤링 App</string>
       </property>
      </widget>
     </item>
     <item row="1" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QComboBox" name="search_type">
         <property name="font">
          <font>
           <family>IM혜민 Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <item>
          <property name="text">
           <string>사용자 ID</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>태그</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="searh_word">
         <property name="font">
          <font>
           <family>IM혜민 Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="inputMask">
          <string/>
         </property>
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="font">
          <font>
           <family>IM혜민 Bold</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>검색</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="2" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QCheckBox" name="Oner_ID">
         <property name="font">
          <font>
           <family>IM혜민 Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>게시자 이름</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="MainText">
         <property name="font">
          <font>
           <family>IM혜민 Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>게시글 내용</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="lover">
         <property name="font">
          <font>
           <family>IM혜민 Bold</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>좋아요 수</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="Hash">
         <property name="font">
          <font>
           <family>IM혜민 Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>해시태그</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="3" column="0" colspan="2">
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <property name="sizeConstraint">
        <enum>QLayout::SetNoConstraint</enum>
       </property>
       <item>
        <widget class="QLabel" name="amount_label">
         <property name="font">
          <font>
           <family>IM혜민 Bold</family>
           <pointsize>12</pointsize>
          </font>
         </property>
         <property name="text">
          <string>데이터 수집 양 :</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="want_amount"/>
       </item>
       <item>
        <widget class="QPushButton" name="minicon_btn">
         <property name="font">
          <font>
           <family>IM혜민 Bold</family>
           <pointsize>12</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>세부조건</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item row="4" column="0" colspan="2">
      <widget class="QTableWidget" name="insta_table">
       <column>
        <property name="text">
         <string>검색어</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>링크</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>사용자이름</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>본문</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>좋아요 수</string>
        </property>
       </column>
       <column>
        <property name="text">
         <string>해시태그</string>
        </property>
       </column>
      </widget>
     </item>
     <item row="5" column="0" colspan="2">
      <widget class="QLabel" name="system_message">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
         <horstretch>1</horstretch>
         <verstretch>50</verstretch>
        </sizepolicy>
       </property>
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>40</height>
        </size>
       </property>
       <property name="sizeIncrement">
        <size>
         <width>0</width>
         <height>0</height>
        </size>
       </property>
       <property name="font">
        <font>
         <family>IM혜민 Bold</family>
         <pointsize>15</pointsize>
        </font>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <property name="text">
        <string>시스템 메세지</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>start_search()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>736</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>645</x>
     <y>21</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>minicon_btn</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>open_minicon()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>782</x>
     <y>139</y>
    </hint>
    <hint type="destinationlabel">
     <x>804</x>
     <y>179</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>start_search()</slot>
  <slot>open_minicon()</slot>
 </slots>
</ui>
